src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -97,6 +104,11 @@ master_create_distributed_table(PG_FUNCTION_ARGS)
 	 */
 	distributedRelation = relation_open(distributedRelationId, AccessExclusiveLock);
 	distributedRelationName = RelationGetRelationName(distributedRelation);
+	distributedRelationSchemaOid = RelationGetNamespace(distributedRelation);
```

Comment:
Nothing actionable, but noting here for future uses of getting qualified schema names. In postgres, there is a function with the signature `generate_qualified_relation_name(Oid relid)`, which is a static function.

 In MX, we copied that into citusruleutils_95.c in this [commit](https://github.com/citusdata/citus-mx/pull/40/commits/bb0ed2d324d17527b04c487f7fc00deb495282e0). We'll probably get that commit to Citus very soon. If we get that function before you guys merge it, I suggest we use it here as well :)